pages/linux/quotacheck.md

```diff
@@ -0,0 +1,28 @@
+# quotacheck
+
+> Scan a filesystem for disk usage, create, check and repair quota files.
+> Should only be run by super-user.
+
+- Check all mounted non-NFS filesystems:
+
+`quotacheck --all`
+
+- Force check even if quotas are enabled:
+
+`quotacheck --force {{filesystem}}`
+
+- Enable debugging mode:
+
+`quotacheck --debug {{filesystem}}`
```

Comment:
Same comment as in the edquota PR: is this state stored somewhere permanent, or only for the current session, or only for the next command, or...?